ui/components/multichain/pages/permissions-page/connection-list-tooltip/connection-list-tooltip.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
```

Comment:
Can we have it typescript one?

Reply:
This may have to be addressed in a future PR. I have a WIP branch based off this branch, so it's really an awkward time to do TS conversion. Definitely afterwards though.

Reply:
Can you add a ticket for this conversion ticket and tackle it after another branch?